cmd/assetsvc/handler.go

```diff
@@ -100,13 +101,14 @@ func min(a, b int) int {
 
 func uniqChartList(charts []*models.Chart) []*models.Chart {
 	// We will keep track of unique digest:chart to avoid duplicates
-	chartDigests := map[string]bool{}
+	chartDigests := map[string]map[string]bool{}
```

Comment:
Mmm, I see your point @andresmgot. I also considered taking this apart in other PR, but since the problem[1] that appeared as a consequence of this change, I just left it in this PR.

But, ok, I  agree, let's revert this change and open another PR for discussing it.

[1] 
```
1. Add bitnami repo
2. Go to bitnami charts -> there are plenty of charts, eg, "bitnami/wordpress"
3. Add jfrog repo
4. Go to bitnami charts -> only 4 charts, there is no longer a "bitnami/wordpress"
```